src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1993,7 +1993,7 @@ internal enum ErrorCode
         WRN_CallerArgumentExpressionParamForUnconsumedLocation = 8966,
         ERR_NewlinesAreNotAllowedInsideANonVerbatimInterpolatedString = 8967,
         ERR_AttrTypeArgCannotBeTypeVar = 8968,
-        // WRN_AttrDependentTypeNotAllowed = 8969, // Backed out of of warning wave 6, may be reintroduced later
+        ERR_RecordStructConstructorCallsDefaultConstructor = 8969, // Added in 17.1 for an invalid C#10 scenario.
```

Comment:
Is reusing the warning number okay here?

Reply:
Maybe, but reusing seems unnecessary. Updated.